src/fps/README.md

```diff
@@ -0,0 +1,80 @@
+# Overview
+
+This new calldata simulation tooling allows developers to write tasks that simulate the state changes that would occur onchain if the proposal were to be executed. The simulator can be run against any mainnet, sepolia, and devnet.
+
+The goal of using this new task tooling for the superchain ops repo is to greatly simplify task development, increase security by reducing errors, reduce sharp edges, and speed development and review of tasks. The simulation is designed to simulate task runs, with all onchain state changes being run locally.
+
+## Task Development and Templates
+
+Developers can now create tasks without writing any Solidity code as long as a predefined template is used. The templates are designed to be as flexible as possible, allowing developers to create tasks that update multiple parameters for a single or super chain. Templates are currently in development and will be released in a future PR.
+
+# Task Configuration File
+
+Each task will have at least 2 config files.
+
+The first file is `taskConfig.toml` which contains the task configuration.
+
+## Task Fields
+
+- `safeAddressString` - the identifier of the safe in the task to run
```

Comment:
These are now not used in the configuration files right? They're added to the solidity.